### ISSUE TITLE
Support for fuel "sub-types"

### ISSUE THREAD

**Issue by niallperks** [akhilrex/hammond#33]
Request for supporting a "sub-type" field for fuel

Each car has a fuel type, but when adding a new fillup it would be nice to have an additional field to define if the fillup used standard or premium fuel.

e.g.
If car is Petrol, possible sub-types are

- Standard
- Premium
- Super Premium

If car is Diesel, possible sub-types are

- Standard
- Premium

For the American market I imagine the octane rating will be used for this

**Comment by akhilrex** [comment]
This has been done. Please rebuild the container once the current build is complete.
